app/build.gradle

```diff
@@ -63,6 +63,8 @@ android {
                         '--add-exports', 'java.base/sun.nio.ch=ALL-UNNAMED',
                         '--add-opens', 'java.base/java.lang.reflect=ALL-UNNAMED',
                         '--add-opens', 'java.base/java.io=ALL-UNNAMED',
+                        '--add-opens', 'java.base/java.net=ALL-UNNAMED',
```

Comment:
is the change needed for this PR?

Reply:
yes, tests do not run for me without it:

```
MjpegScreenshotTest > shouldReceiveDataFromMjpegServer FAILED
    java.lang.reflect.InaccessibleObjectException: Unable to make static java.lang.String java.net.URL.toLowerCase(java.lang.String) accessible: module java.base does not "opens java.net" to unnamed module @79e4c792
        at java.base/java.lang.reflect.AccessibleObject.checkCanSetAccessible(AccessibleObject.java:354)
        at java.base/java.lang.reflect.AccessibleObject.checkCanSetAccessible(AccessibleObject.java:297)
        at java.base/java.lang.reflect.Method.checkCanSetAccessible(Method.java:199)
        at java.base/java.lang.reflect.Method.setAccessible(Method.java:193)
        at org.powermock.reflect.internal.WhiteboxImpl.doGetAllMethods(WhiteboxImpl.java:1508)
        at org.powermock.reflect.internal.WhiteboxImpl.getAllMethods(WhiteboxImpl.java:1482)
        at org.powermock.reflect.internal.WhiteboxImpl.getMethods(WhiteboxImpl.java:1750)
        at org.powermock.reflect.internal.WhiteboxImpl.getMethods(WhiteboxImpl.java:1789)
        at org.powermock.reflect.internal.WhiteboxImpl.getBestMethodCandidate(WhiteboxImpl.java:1008)
        at org.powermock.core.MockInvocation.findMethodToInvoke(MockInvocation.java:58)
        at org.powermock.core.MockInvocation.init(MockInvocation.java:35)
        at org.powermock.core.MockInvocation.<init>(MockInvocation.java:22)
        at org.powermock.core.MockGateway.doMethodCall(MockGateway.java:155)
        at org.powermock.core.MockGateway.methodCall(MockGateway.java:138)
        at io.appium.uiautomator2.server.mjpeg.MjpegScreenshotTest.shouldReceiveDataFromMjpegServer(MjpegScreenshotTest.java:51)
```